### PR TITLE
Added the second map

### PR DESCRIPTION
**Pull request by DatOnePlayer**
Please review and leave me your thoughts

### CODE REVIEW

File: content/zones/frozen-lakes.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  zone: frozen-lake
+}
+requirements: [
+  {
+    type: Launched
+    zone: derelict-desert-outpost
+  }
+  {
+    type: unlock
+    block: iron-conveyor.json
```
Comment (JerichoFletcher) [inline content/zones/frozen-lakes.json:19]: Why do you put .json after iron-conveyor here?

Comment (DatOnePlayer) [reply content/zones/frozen-lakes.json:19]: Lol sorry, can you fix that for me? I have some urgencies to do

Comment (DatOnePlayer) [reply content/zones/frozen-lakes.json:19]: Or do you want me to do it a bit later

Comment (JerichoFletcher) [reply content/zones/frozen-lakes.json:19]: Aight I'll fix it

**Pull request merged**